api/jobs.yaml

```diff
@@ -116,6 +131,45 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    required:
+      - required
+    properties:
+      required:
```

Comment:
nit: What do you think about naming this `is_required` since it's a boolean?

Reply:
I think that makes sense, switched.